src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -109,6 +109,19 @@ public string Url
 
         internal string LastServerVersion;
 
+        public TimeSpan SendTimeout
```

Comment:
This is TCP related. I do not see a reason for us having that in RequestExecutor

Reply:
We need a timeout configuration in the RequestExecutor for TCP - and we need it BEFORE we initialize the TcpClient

Reply:
Still it is not related in any way to RequestExecutor. Maybe it can be moved to conventions, but only place where this is used in Client API is subscriptions and I'm not sure if this is something that we want to have configurable in the Client API